contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+and on node `gke-cstor-it-default-pool-2`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-mirrored
+spec:
+  type: mirrored
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        // System generated
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-2
+        // System generated
+        id: pool-1-bd2
+        // System generated
+        capacity: 12345234
+
+  - hostName: gke-cstor-it-default-pool-2
+
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-2-bd-1
+        // System generated
+        id: pool-2-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-2-bd-2
+        // System generated
+        id: pool-2-bd2
+        // System generated
+        capacity: 12345234
+
+```
+
+Expanding mirrored pools -- the spec will look
+following:
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-mirrored
+spec:
+  type: mirrored
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-2
+        id: pool-1-bd2
+        // System generated
+        capacity: 12345234
+
+    // New block device group added
+    - groupName: group2
+      blockDevices:
+      - blockDeviceName: pool-1-bd-3
+        id: pool-1-bd3
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-4
+        id: pool-1-bd4
+        // System generated
+        capacity: 12345234
+
+  - name: gke-cstor-it-default-pool-2
+
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-2-bd-1
+        // System generated
+        id: pool-2-bd1
+        // System generated
+        capacity: 12345234
+      - blockDeviceName: pool-2-bd-2
+        // System generated
+        id: pool-2-bd2
+        // System generated
+        capacity: 12345234
+
+    // New block device group added
+    - groupName: group2
+      blockDevices:
+      - blockDeviceName: pool-2-bd-3
+        // System generated
+        id: pool-2-bd3
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-2-bd-4
+        // System generated
+        id: pool-2-bd4
+        // System generated
+        capacity: 12345234
+
+```
+
+**NOTE:**
+In case of striped pool expansion no new group but block device in the present
+group as striped has only one groups but in all other raid groups a new group is
+added for expansion.
+
+### Operations Workflow
+( Includes Pool Expansion, Pool Deletion, Block Device Offline, Block Device Replacment and Pool Migration)
```

Comment:
Try to run this doc via a spell check. There are some spelling issues.

Reply:
Done. Ran via a spell checker. Thanks.